README.md

```diff
@@ -52,6 +52,10 @@ optionally (though the build-system might currently require these libraries):
 - libgbm-dev
 - libclang-dev
 - libpipewire-0.3-dev
+- librust-atk-dev
+- libgtk-3-dev
```

Comment:
Would be good to eliminate this dependency from the project that's depending on it.

Reply:
https://github.com/pop-os/cosmic-edit/issues/11

Reply:
Purged